status: Add 2024q3 FreeBSD Foundation report entry (#431)
freebsd/freebsd-doc: website/content/en/status/report-2024-07-2024-09/freebsd-foundation.adoc

link:https://www.freebsd.org/projects/summerofcode/[FreeBSD completed our 20th consecutive year participating in Google Summer of Code].
The link:https://wiki.freebsd.org/SummerOfCode2024Projects[11 projects for this summer] are now complete; nine passed.

The Foundation has been providing project management support for the FreeBSD Open Container Initiative (OCI) Working Group, with Alice Sowerby hosting the bi-weekly meeting, and running the recent link:https://github.com/oci-playground/freebsd-podman-testing/blob/main/README.md[Podman on FreeBSD testing project].
The link:https://opencontainers.org/[OCI] develops open industry standards for cloud native container formats and runtimes, ensuring platform consistency.
The FreeBSD OCI Working Group is defining these standards for FreeBSD, with implementations using jails and potentially lightweight VMs with bhyve.
Refer to the Foundation's link:https://freebsdfoundation.org/project/oci-container-support/[OCI Container Support Project page] for details.

In other Foundation news:

* Isaac Freund joined the Foundation's development team as a Userland Developer.

>   As the lead developer of the River Wayland Compositor and a member of the Core Zig Team, we're excited about the experience Isaac will be bringing to FreeBSD.

Reviewer: ```suggestion
  As the lead developer of the River Wayland Compositor and a member of the Core Zig Team, we are excited about the experience Isaac will be bringing to FreeBSD.
```

  Refer to the <<_audio_stack_improvements,Audio Stack Improvements>> entry for the latest news.

* Olivier Certner has two entries in this report.
  You can read about his latest work in the <<_scheduling_priorities_256_queue_runqueues_sub_project,Scheduling Priorities: 256-queue Runqueues Sub-Project>> and <<_mac_do4_setcred2_mdo1,mac_do(4), setcred(2), mdo(1)>> report entries.

* Bjoern Zeeb continued to improve wireless networking on FreeBSD.
  You can read the latest news in Bjoern's <<_wireless_update,Wireless Update>> entry.

* Philip Paeps continued work on a contract to modernize the FreeBSD cluster.

* Chih-Hsin Chang has continued work to port OpenStack components so that the cloud computing platform can be run on FreeBSD hosts.
  Refer to the <<_openstack_on_freebsd,OpenStack on FreeBSD>> entry for the latest information.

* Other members of the Foundation's technology team contributed to FreeBSD development efforts.  For example:
** Mitchell Horne committed work for RISC-V, including adding support for the Supervisor-mode: Page-Based Memory Types (Svpbmt) extension

> ** Ed Maste removed the deprecated mergmaster tool in favor man:etcupdate[8] for updating files not managed by install world

Reviewer: ```suggestion
** Ed Maste removed the deprecated mergmaster tool in favor of man:etcupdate[8] for updating files not managed by install world
```

> ** Joe Mingrone updated our base libpcap and tcdump

Reviewer: ```suggestion
** Joe Mingrone updated our base libpcap and tcpdump
```

Or even better
```suggestion
** Joe Mingrone updated our base libpcap and man:tcpdump[1]
```

** Li-Wen Hsu kept our Jenkins port tracking the latest upstream versions with a number of port updates.

==== Continuous Integration and Workflow Improvement
As part of our continued support of the FreeBSD Project, the Foundation supports a full-time staff member dedicated to improving the Project's continuous integration system and test infrastructure.

==== Advocacy

During the third quarter of 2024, we continued growing our efforts to drive awareness, advocate for the project, highlight users, and bring educational content to the FreeBSD community.
Below are some of those efforts.

* Presented at the EuroBSDcon 2024 FreeBSD Developer Summit.
  Slides and the Live stream are https://wiki.freebsd.org/DevSummit/202409[now available].
* Attended and exhibited at https://2024.eurobsdcon.org/[EuroBSDCon 2024].
  The Foundation was again a Silver Sponsor.
* Finalized our Bronze Sponsorship of the https://freebsdfoundation.org/news-and-events/event-calendar/openzfs-developer-summit-2024/[OpenZFS User and Developer Summit]

> * Began planning the https://freebsdfoundation.org/news-and-events/event-calendar/fall-2024-freebsd-summit/[Fall 2024 FreeBSD Summit] taking place November 7-8, 2024 in San Jose, CA. The program is now available and registration is open.

Reviewer: ```suggestion
* Began planning the https://freebsdfoundation.org/news-and-events/event-calendar/fall-2024-freebsd-summit/[Fall 2024 FreeBSD Summit] taking place November 7-8, 2024 in San Jose, CA.
 The program is now available and registration is open.
```

** https://freebsdfoundation.org/news-and-events/latest-news/apples-open-source-roots-the-bsd-heritage-behind-macos-and-ios/[Apple's Open Source Roots: The BSD Heritage Behind macOS and iOS]
* Published the https://freebsdfoundation.org/news-and-events/newsletter/july-2024-foundation-update/[July 2024], https://freebsdfoundation.org/news-and-events/newsletter/august-2024-foundation-update/[August 2024], and https://freebsdfoundation.org/news-and-events/newsletter/september-newsletter/[September 2024] FreeBSD Foundation Updates.
* Released the https://freebsdfoundation.org/our-work/journal/browser-based-edition/configuration-management-2/[May/June 2024] and https://freebsdfoundation.org/our-work/journal/browser-based-edition/storage-and-filesystems/[July/August 2024] issues of the FreeBSD Journal with HTML versions of the articles.

==== Legal/FreeBSD IP

The Foundation owns the FreeBSD trademarks, and it is our responsibility to protect them.
We also provide legal support for the core team to investigate questions that arise.

> Go to link:https://freebsdfoundation.orgr[] to find more about how we support FreeBSD and how we can help you!

Reviewer: ```suggestion
Go to link:https://freebsdfoundation.org[] to find more about how we support FreeBSD and how we can help you!
```
